<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Calorie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pj_calorie">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Note:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pj_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pj_note">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pj_prix">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="pj_description">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>191</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retourBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="pj_picture">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>picture to be put here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
